Source/WebKit/Shared/WebCoreArgumentCoders.cpp

```diff
@@ -262,9 +262,9 @@ void ArgumentCoder<RectEdges<bool>>::encode(Encoder& encoder, const RectEdges<bo
     SimpleArgumentCoder<RectEdges<bool>>::encode(encoder, boxEdges);
 }
     
-bool ArgumentCoder<RectEdges<bool>>::decode(Decoder& decoder, RectEdges<bool>& boxEdges)
+std::optional<RectEdges<bool>> ArgumentCoder<RectEdges<bool>>::decode(Decoder& decoder)
```

Comment:
Note (not related to your patch):
since we are changing this, we could just change this to the correct simpleargumentcoder form, e.g. inheriting from it.

Also Note (not related to your patch):
this encodes bools as data, which highlights the futility of !!bool thing in bool encoder that makes the bool encoder more complex than it needs.